src/sentry/static/sentry/app/components/broadcastModal.jsx

```diff
@@ -14,7 +14,7 @@ const ReleaseAnnouncement = ({close}) => {
         <span className="badge badge-square badge-new">NEW</span>
       </h3>
       <p>
-        <img src={mediaUrl + 'dist/release-commits-modal.gif'} />
+        <img src={mediaUrl + 'images/onboarding/release-commits-modal.gif'} />
```

Comment:
The correct solution will be to figure out why it's not being moved into the correct folder like every other image.